lib/build-scss.js

```diff
@@ -129,83 +133,50 @@ const compileAndWriteStyleSheets = ({
           isDefaultThemeVariant,
         });
       }
+
       fs.writeFileSync(`${outDir}/${paragonThemeOutputFilename}`, `${JSON.stringify(paragonThemeOutput, null, 2)}\n`);
+    })
+    .then(() => {
+      ora().succeed(chalk.underline.bold.green(`Stylesheets for ${capitalize(name)} build successfully!`));
+      // eslint-disable-next-line no-console
+      console.log();
+    })
+    .catch((error) => {
+      ora().fail(chalk.bold(`Failed to build stylesheets for ${capitalize(name)}: ${error.message}`));
     });
 };
 
-program
-  .version('0.0.1')
-  .description('CLI to compile Paragon\'s core and themes\' SCSS into CSS.')
-  .addOption(
-    new Option(
-      '--corePath <corePath>',
-      'Path to the theme\'s core SCSS file, defaults to Paragon\'s core.scss.',
-    ),
-  )
-  .addOption(
-    new Option(
-      '--themesPath <themesPath>',
-      `Path to the directory that contains themes' files. Expects directory to have following structure:
-      themes/ 
-        light/
-        │  ├─ index.css
-        │  ├─ other_css_files
-        dark/
-        │  ├─ index.css
-        │  ├─ other_css_files
-        some_other_custom_theme/
-        │  ├─ index.css
-        │  ├─ other_css_files
-        ...
-      
-      where index.css has imported all other CSS files in the theme's subdirectory. The script will output
-      light.css, dark.css and some_other_custom_theme.css files (together with maps and minified versions).
-      You can provide any amount of themes. Default to paragon's themes.
-      `,
-    ),
-  )
-  .addOption(
-    new Option(
-      '--outDir <outDir>',
-      'Specifies directory where to out resulting CSS files.',
-    ),
-  )
-  .addOption(
-    new Option(
-      '--defaultThemeVariants <defaultThemeVariants...>',
-      `Specifies default theme variants. Defaults to a single 'light' theme variant.
-      You can provide multiple default theme variants by passing multiple values, for
-      example: \`--defaultThemeVariants light dark\`
-      `,
-    ),
-  );
-
-program.parse(process.argv);
-
-const options = program.opts();
-const {
-  corePath = path.resolve(__dirname, 'styles/scss/core/core.scss'),
-  themesPath = path.resolve(__dirname, 'styles/css/themes'),
-  outDir = './dist',
-  defaultThemeVariants = ['light'],
-} = options;
+/**
+ * Builds SCSS stylesheets based on the provided command arguments.
+ *
+ * @param {Array<string>} commandArgs - Command line arguments for building SCSS stylesheets.
+ */
+function buildScssCommand(commandArgs) {
+  const argv = minimist(commandArgs);
+  const corePath = argv.corePath || path.resolve(process.cwd(), 'styles/scss/core/core.scss');
+  const themesPath = argv.themesPath || path.resolve(process.cwd(), 'styles/css/themes');
+  const outDir = argv.outDir || './dist';
+  const defaultThemeVariants = argv.defaultThemeVariants ? argv.defaultThemeVariants.split(',') : ['light'];
```

Comment:
Reading through minimist docs I found that you can provide default values to args, I think we should try doing that, right now it looks a little dirty

this seems to work for me locally (also, I don't think you need to split `defaultThemeVariants`)
```js
function buildScssCommand(commandArgs) {
  const defaultArgs = {
    corePath: path.resolve(process.cwd(), 'styles/scss/core/core.scss'),
    themesPath: path.resolve(process.cwd(), 'styles/css/themes'),
    outDir: './dist',
    defaultThemeVariants: 'light',
  };

  const {
    corePath,
    themesPath,
    outDir,
    defaultThemeVariants,
  } = minimist(commandArgs, { default: defaultArgs });
```

Reply:
Thanks